### DISCUSSION TITLE
Paket Bulanan

### DISCUSSION THREAD

**Discussion by asdfok** [hotspotbilling/phpnuxbill#25] [Ideas]
Hi, bisa tolong ditambah untuk tipe paketnya? Saya masih memakai sistem simple Queue. jd beberapa pelanggan rumahan diberi IP statik, & akses hotspot untuk 4 perangkat.
Jadi untuk isolir user, cukup dengan menonaktifkan nama dari "Simple Queue"nya.

**Comment by ibnux** [comment]
Karena Banyak yang harus dirubah.
Tidak bisa dikerjakan
